contributed_definitions/NXsts.nxdl.xml

```diff
@@ -889,13 +932,12 @@ x
 y
 z
 single point default plot # current(I) vs bias(V)-->
-            <group name="single_point" type="NXdata" optional="true"/>
-            <!--line scan default plot # multi I vs. V curves-->
-            <group name="line_scan" type="NXdata" optional="true"/>
-            <!--alternative plot # current(I) curve in the 2D space of (position(x), bias(V))-->
-            <group name="alternative_plot" type="NXdata" optional="true"/>
-            <!--mesh scan default plot # 2D slices of the above alternative plot-->
-            <group name="mesh_scan" type="NXdata" optional="true"/>
-        </group>
+        <group name="single_point" type="NXdata" optional="true"/>
```

Comment:
these NXdata groups do not come with documentation. Note that comments are only for developers, but those who use the Appdef will not see and cannot understand the purpose of these groups.

Reply:
I added some documentation for these `NXdata`. But, I think @Souikey or @ca-palma must verify them.